atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -447,15 +495,28 @@ private void setUpForLocalServices() throws IOException {
                     .withBody(("\"" + localPingableLeader.getUUID().toString() + "\"").getBytes())));
             return null;
         }).when(environment).accept(isA(PingableLeader.class));
-        setupLeaderBlockInConfig();
+        setUpLeaderBlockInConfig();
     }
 
     private void setUpForRemoteServices() throws IOException {
         availableServer.stubFor(LEADER_UUID_MAPPING.willReturn(aResponse().withStatus(404)));
-        setupLeaderBlockInConfig();
+        setUpLeaderBlockInConfig();
+    }
+
+    private void setUpTimeLockBlockInInstallConfig() {
+        when(config.timelock()).thenReturn(Optional.of(mockClientConfig));
+    }
+
+    private void setUpTimeLockBlockInRuntimeConfig() {
+        when(runtimeConfig.timelockRuntime()).thenReturn(Optional.of(mockTimelockRuntimeConfig));
```

Comment:
nit: `mockTimelockRuntimeConfig` is not a mock. Also, might be a good idea to inline this variable.

Reply:
Ack - renamed. I think this was a hangover from a previous version where I tried to use mocks.